frontend/src/components/widgets/Slider/Slider.tsx

```diff
@@ -183,24 +186,52 @@ class Slider extends React.PureComponent<Props, State> {
     $thumbIndex: number
     $value: any
   }): JSX.Element => {
-    const thumbValueStyle = sliderOverrides.ThumbValue.style({
-      $disabled: this.props.disabled,
-    }) as React.CSSProperties
+    const { disabled, theme } = this.props
+    const { colors, fonts, fontSizes, lineHeights } = theme
+    const thumbValueStyle: React.CSSProperties = {
+      fontFamily: fonts.mono,
+      fontSize: fontSizes.smDefault,
+      paddingBottom: fontSizes.twoThirdSmDefault,
+      color: disabled ? colors.gray : colors.primary,
+      top: "-22px",
+      position: "absolute",
+      whiteSpace: "nowrap",
+      backgroundColor: colors.transparent,
+      lineHeight: lineHeights.base,
+      fontWeight: "normal",
+    }
 
     return (
-      <div style={thumbValueStyle}>
+      <div style={thumbValueStyle} data-testid="stThumbValue">
```

Comment:
This could be a styled component since it's just pure html/react. It could be used when we move away from baseweb potentially